recipe/meta.yaml

```diff
@@ -32,15 +34,22 @@ about:
   license_file: COPYING
   summary: Basic astronomical computations for Python
   description: |
-    "PyEphem provides basic astronomical computations for the Python programming
-    language. Given a date and location on the Earth's surface, it can compute
-    the positions of the Sun and Moon, of the planets and their moons, and of
-    any asteroids, comets, or earth satellites whose orbital elements the user
-    can provide."
+    PyEphem provides an ``ephem`` Python package for performing high-precision
+    astronomy computations. The underlying numeric routines are coded in C and
+    are the same ones that drive the popular `XEphem`_ astronomy application,
+    whose author, Elwood Charles Downey, generously gave permission for their
+    use in PyEphem. The name *ephem* is short for the word *ephemeris*, which
+    is the traditional term for a table giving the position of a planet,
+    asteroid, or comet for a series of dates. The design of PyEphem emphasizes
+    convenience and ease of use. Both celestial bodies and the observer's
+    location on Earth are represented by Python objects, while dates and
+    angles automatically print themselves in standard astronomical formats.
   doc_url: http://rhodesmill.org/pyephem/
   dev_url: https://github.com/brandon-rhodes/pyephem
 
 extra:
   recipe-maintainers:
+    - bsipocz
+    - mwcraig
```

Comment:
@bsipocz and @mwcraig I added you two here b/c you where in `pyephem`, which we just archived, and this one is the same package. Is that OK?

Reply:
👍 
Feel free to add me to add me to any astropy affiliated packages (full, but usually growing list is here: http://www.astropy.org/affiliated/registry.json), and also to the ones any of the astropy related packages depend on.

Reply:
:+1: (and everything else @bsipocz said too)